src/main/java/edu/ie3/datamodel/io/sink/InfluxDbSink.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.sink;
+
+import edu.ie3.datamodel.exceptions.SinkException;
+import edu.ie3.datamodel.io.FileNamingStrategy;
+import edu.ie3.datamodel.io.connectors.InfluxDbConnector;
+import edu.ie3.datamodel.io.extractor.NestedEntity;
+import edu.ie3.datamodel.io.processor.ProcessorProvider;
+import edu.ie3.datamodel.io.processor.timeseries.TimeSeriesProcessorKey;
+import edu.ie3.datamodel.models.UniqueEntity;
+import edu.ie3.datamodel.models.input.container.JointGridContainer;
+import edu.ie3.datamodel.models.result.ResultEntity;
+import edu.ie3.datamodel.models.timeseries.TimeSeries;
+import edu.ie3.datamodel.models.timeseries.TimeSeriesEntry;
+import edu.ie3.datamodel.models.value.Value;
+import java.time.ZonedDateTime;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.influxdb.InfluxDB;
+import org.influxdb.dto.BatchPoints;
+import org.influxdb.dto.Point;
+
+/** InfluxDB Sink for result and time series data */
+public class InfluxDbSink implements DataSink {
+  private static final Logger log = LogManager.getLogger(InfluxDbSink.class);
+  /** Field name for timestamp field in result entities */
+  private static final String FIELD_NAME_TIMESTAMP = "timestamp";
+  /** Field name for timestamp field in time series */
+  private static final String FIELD_NAME_TIME = "time";
+  /** Field name for input model uuid field in result entities */
+  private static final String FIELD_NAME_INPUT = "inputModel";
```

Comment:
This part is related to #91. Different places within the code use information about how certain fields are named. This information is duplicated and needs some refactoring to have one single point of truth for field names throughout the whole code.

This place should then also be addressed with PR #91

Reply:
At least one field may not have that single point of truth: InfluxDB requires a "time" column and this column can not be renamed, only aliased. But most of our timestamp fields are called "timestamp" or "datum". But #91 is still applicable for the other field names, however, I would resolve this in 91 and not this PR.